algorithmic_efficiency/workloads/librispeech/librispeech_pytorch/input_pipeline.py

```diff
@@ -29,15 +30,20 @@ def __len__(self):
     return self.sample_size
 
   def pad_collate(self, batch):
-    max_input_len = float("-inf")
-    max_target_len = float("-inf")
+    max_input_len = 0
+    max_target_len = 0
 
+    # Get maximum lenght of entire batch
```

Comment:
Nitpick: Could you fix the typo of "length"

Reply:
Good catch! I've fixed both.